src/vm/node_modules/VM.js

```diff
@@ -6396,7 +6396,7 @@ function buildFilesystemZonecfg(vmobj, payload, options)
     // remove is a list of disk paths, add a remove for each now.
```

Comment:
nit, can we fix or remove this comment please?

Reply:
Done